crypto/signing/mcl/suiteBLS12_381.go

```diff
@@ -146,14 +126,12 @@ func (s *SuiteBLS12) IsInterfaceNil() bool {
 
 // BaseG1 returns the generator point for G1
 func BaseG1() string {
-	v := basePointG1Str.Load()
-	vStr := v.(string)
-	return vStr
+	doInit.Do(blsInit)
+	return g1str
 }
 
 // BaseG2 returns the generator point for G2
 func BaseG2() string {
-	v := basePointG2Str.Load()
-	vStr := v.(string)
-	return vStr
+	doInit.Do(blsInit)
```

Comment:
I think you can remove from here the init

Reply:
done